src/js/fee.js

```diff
@@ -125,10 +125,7 @@
 
       if (content !== 'raw') {
         returnContent = returnContent.replace(/<p>(?:<br ?\/?>|\u00a0|\uFEFF| )*<\/p>/g, '<p>&nbsp;</p>');
-        returnContent = returnContent.replace(/<p>\[/g, '[');
-        returnContent = returnContent.replace(/\]<\/p>/g, ']');
-        returnContent = returnContent.replace(/<br \/>/g, '');
-        returnContent = returnContent.replace(/<p>&nbsp;<\/p>/g, '<br />');
+        returnContent = window.switchEditors.pre_wpautop(returnContent);
```

Comment:
ok, so it sounds like its an improvement on the existing filtering. The filtering will always be an issue we have to keep an eye on.